source/articles/2015/how-to-generate-sales-leads-as-a-freelance-developer.md

```diff
@@ -0,0 +1,223 @@
+
+author: Adarsh Pandit
+category: consulting
+date: 2015-08-08
+logo: terminal_icon.png
+published: true
+tags: [consulting, sales, leads, networking, content marketing]
+---
+
+
+As I wrote in [this post][1],
+I'm sharing what I know about
+consulting in hopes it's helpful to others.
+
+This post focuses on how to generate sales leads or
+"finding people who are interested in paying you."
+
+Everything here I learned as a freelance
+management consultant,
+while selling millions of dollars of work
+for a medium-sized boutique firm,
+and now as a freelance developer.
+
+[1]: /how-to-run-a-freelance-software-consulting-business
+
+## Preface
+
+It's a weird thing to hire someone else
+and pay them for their time.
+
+The relationship requires trust above all else.
+Once one party loses trust in the other,
+it's best to move on, re-evaluate what happened,
+and find other work.
+As such, everything you do,
+needs to be authentic, honest, and professional,
+including trying to drum up business.
+
+Also it's worth noting these tactics work best
+when you do them very consistently
+for a long time.
+There's no substitute for building good routines
+and doing the work.
+
+Finally, it should be noted
+that I am by no means perfect at sales
+and your mileage may vary.
+
+Okay lets start:
+
+## Networking
+
+### Specific Tactics I Use
+
+* Keep a list of people to stay in touch with.
+* Be sure to follow up regularly. Use tools to automate the process.
+  I like to send a reminder to [`every2months@followup.cc`][2] with
+  the subject line "Get Coffee With PERSON" and get reminded regularly.
+  Repeating calendar invites are great for this as well.
+* Be Helpful 1 - Make introductions on behalf of people you think might
+  benefit from meeting each other.
+* Be Helpful 2 - Forward articles/videos/tweets/etc to people if you think
+  they'll find it interesting / relevant.
+* Don't spend time with people you don't like.
+* Send them articles when you think they may find them interesting. [Here's an
+  old post where I talk about doing it with Google Reader][reader post], which
+tells you how old the post is.
+
+[2]: http://help.followup.cc/knowledge_base/topics/cheat-sheet
+[reader post]: /google-reader-as-a-networking-tool
+
+The goal of these activities
+is to make "business friends"
+whom you like, and who like you.
+If you don't like someone,
+or are just hanging out with them
+because you feel you have to,
+it won't work.
+
+You should be helpful to them,
+recommending candidates to hire,
+articles to read,
+people to meet,
+and services to try.
+
+You should ask them for advice frequently.
+You should ask them for help rarely.
+You should support their businesses,
+retweet their product announcements,
+and support their crowdfunding campaigns,
+_but only do so because you feel like it_.
+
+I should note
+that every job I've ever gotten
+has been a result of networking.
+I'm guessing this is fairly broadly true.
```

Comment:
Good call. Done in af18f04



-- 
@adarshp <http://www.twitter.com/adarshp>
313.454.1515 (mobile)
See when I am free: http://bit.ly/adarshcal

On Fri, Oct 30, 2015 at 1:56 PM, Chris Eigner <notifications@github.com>
wrote:

> In
> source/articles/2015/how-to-generate-sales-leads-as-a-freelance-developer.md
> <https://github.com/adarsh/blog/pull/15#discussion_r43551259>:
>
> > +recommending candidates to hire,
> > +articles to read,
> > +people to meet,
> > +and services to try.
> > +
> > +You should ask them for advice frequently.
> > +You should ask them for help rarely.
> > +You should support their businesses,
> > +retweet their product announcements,
> > +and support their crowdfunding campaigns,
> > +_but only do so because you feel like it_.
> > +
> > +I should note
> > +that every job I've ever gotten
> > +has been a result of networking.
> > +I'm guessing this is fairly broadly true.
>
> Back to back *ly words sounds funny. Maybe drop the "fairly".
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/adarsh/blog/pull/15/files#r43551259>.
>